distribution/lib/Standard/Table/0.0.0-dev/src/Data/Table.enso

```diff
@@ -2701,6 +2804,12 @@ concat_columns column_set all_tables result_type result_row_count on_problems =
         sealed_storage = storage_builder.seal
         Column.from_storage column_set.name sealed_storage
 
+## PRIVATE
+   A helper that creates a two-column table from a map.
+map_to_lookup_table : Map Any Any -> Text -> Text -> Table
+map_to_lookup_table map key_column value_column =
+    Table.new [[key_column, map.keys], [value_column, map.values]]
```

Comment:
I'm wondering if we can rely on `keys` and `values` always being ordered consistently.

I guess we should be. But I don't think _technically_ we have such a guarantee - I guess the ordering _could_ be arbitrary.

Maybe that could be safer?
```suggestion
    keys_and_values = map.to_vector
    Table.new [[key_column, keys_and_values.map .first], [value_column, keys_and_values.map .second]]
```

Reply:
Done.